commands/import.go

```diff
@@ -88,8 +94,6 @@ func (i *Importer) parseArgs() error {
 		return cli.NewExitError(err.Error(), -1)
 	}
 
-	i.res = resources.InitResources(i.configFile)
```

Comment:
Moved into the main run function since this isn't parsing. This will end up dialing the database, etc.